### PR TITLE
Fix imports path

### PR DESCRIPTION
**Pull request by foxdalas**
Some fixes for simple go build .

### CODE REVIEW

File: services/http.go [hunk 1]
```diff
@@ -2,13 +2,14 @@ package services
 
 import (
 	"fmt"
+	"github.com/snail007/goproxy/utils"
```
Comment (snail007) [inline services/http.go]: import repeated

Comment (foxdalas) [reply services/http.go]: Thx.

Comment (foxdalas) [reply services/http.go]: Fixed

**Comment by foxdalas** [comment]
Fixed

**Pull request merged**